File: 1672-richest-customer-wealth/1672-richest-customer-wealth.go
var v [51]int
var acc *[][]int
var wg sync.WaitGroup
var n int
func test(idx int) {
    // fmt.Println((*acc)[idx][0])
    ans := 0
    for i:=0; i<n;i++ {
        ans += (*acc)[idx][i]
    }
    v[idx] = ans
}

func maximumWealth(accounts [][]int) int {
    acc = &accounts
    m := len(accounts)
    n = len(accounts[0])
    ans := 0
    for i := 0; i<m; i++ {
        wg.Add(1)
        go func(i int) {
            test(i)
            wg.Done()
        }(i)
    }
    wg.Wait()
    for i:=0; i<m;i++ {
        if v[i] > ans {
            ans = v[i]
        }
    }
    return ans
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int ans = 0;
        const int m = accounts.size();
        if(m == 0) return 0;
        const int n = accounts[0].size();
        for(int i=0;i<m;++i) {
            int sum = 0;
            for(int j=0;j<n;++j) {
                sum += accounts[i][j];
            }
            ans = max(ans, sum);
        }
        return ans;
    }
};